build/embed/integrations.mk

```diff
@@ -8,23 +8,26 @@ PKG_DIR_BIN_OHIS    	?= $(NRI_PKG_DIR)/var/db/newrelic-infra/newrelic-integratio
 OHI_ARCH ?= amd64
 OHI_OS   ?= linux
 
+# FIPS Suffix
+FIPS_SUFFIX ?= $(if $(FIPS),-fips,)
```

Comment:
the env variable called FIPS is already including the suffix, we should be doing something similar to what is done HERE https://github.com/newrelic/infrastructure-agent/pull/1942/files#diff-5f1a71e18a8f7a68293b9af4835b00650c67c5bc81776c7e929555cea5cc09c9

Reply:
I thought it made sense to have this condition here explicitly because we can clearly understand which package will be downloaded without seeing the workflows and env vars. But I think it can be removed if we always ensure either `-fips` or `` will be sent in the `FIPS` var.